test/org/sagebionetworks/bridge/services/ScheduledActivityServiceMockTest.java

```diff
@@ -117,6 +121,11 @@ public void before() {
         SurveyResponseService surveyResponseService = mock(SurveyResponseService.class);
         when(surveyResponseService.createSurveyResponse(
             any(GuidCreatedOnVersionHolder.class), anyString(), any(List.class), anyString())).thenReturn(surveyResponse);
+
+        Subpopulation subpopulation = Subpopulation.create();
+        
+        SubpopulationService subpopService = mock(SubpopulationService.class);
+        when(subpopService.getSubpopulations(any())).thenReturn(Lists.newArrayList(subpopulation));
```

Comment:
For completeness, can we have this return multiple subpopulations, then verify that we correctly use the earliest consent?